src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
> The function is called with the request options object as its parameter. It can **modify this object** or return a new one.

My idea is that in most cases the user will just want to modify the existing object (like in the unit test). Then nothing is needed to be returned.

```js
requestTransform: function(options)
{
    options.qs = {
        foo: "bar"
    };
}
```

But if the user wants to substitute the object with a new one, `return` can be used for this. Do you think this is error-prone?